galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.ceci.manager.ivt/src/main/java/dev/galasa/cicsts/ceci/manager/ivt/CECIManagerIVT.java

```diff
@@ -75,6 +80,22 @@ public void testNotNull() throws CicstsManagerException  {
       assertThat(batch).isNotNull();
    } 
    
+   /**
+    * Starts the CECI session.
+    * 
+    * @throws CicstsManagerException 
+    * @throws CeciException 
+    */
+   @Test
+   public void testStartCeciSession() throws CeciException, CicstsManagerException {
+	   logger.info("Testing starting a CECI session");
+	   cics.ceci().startCECISession(ceciTerminal);
+	   
+	   logger.info("If the CECI session started successfully, the terminal should be able to process a CECI command");
```

Comment:
Is there a simpler way to check - maybe by parsing the screen content?

Reply:
The startCECISession method does this internally and should throw an exception if the CECI initial screen isn't shown. Line 94 onwards is just an extra check

Reply:
True - but our validation could be simpler maybe?

Reply:
I see what you mean - I have added a simpler validation, will push asap